src/Model/NewsCategoryModel.php

```diff
@@ -137,21 +137,25 @@ public static function findPublishedByArchives(array $archives, array $ids = [])
      */
     public static function findPublishedByIdOrAlias($idOrAlias)
     {
-        $values = [$idOrAlias];
+        $values = [];
+        $columns = [];
         $t = static::getTableAlias();
 
         // Determine the alias condition
-        if (MultilingualHelper::isActive()) {
-            $aliasCondition = 't1.alias=? OR t2.alias=?';
-            $values[] = $idOrAlias;
+        if (is_numeric($idOrAlias)) {
+            $columns[] = "$t.id=?";
             $values[] = $idOrAlias;
```

Comment:
Shouldn't we force integer here? What do you think?

Reply:
Yes, that certainly would not hurt.